CHANGELOG.md

```diff
@@ -1,3 +1,100 @@
+## 2023-09-14 v0.20.0
+
+> Note: IP address 0.0.0.0 is no longer advertised by a node
+
+> CLI option `store-resume-peer` has been removed
+
+## What's Changed
+
+Release highlights:
+* RLN is now part of standard release (is no longer EXPERIMENTAL feature)
+* Interop tests between nwaku and js-waku are now gating PRs and releases
+* Libwaku has been made more threadsafe (1 out of 3 improvements applied.)
+* Added autosharding option on various protocol APIs
+* TODO: @rymnc https://github.com/waku-org/nwaku/pull/1966
```

Comment:
@vpavlin here we need to write that the `rln-relay-membership-group-index` flag has been removed, along with `rln-relay-pubsub-topic` and `rln-relay-content-topic`